ruby.rb

```diff
@@ -0,0 +1,58 @@
+Puppet::Type.type(:validate_db_connection).provide(:ruby) do
+
+  commands :psql => 'psql'
+
+  def psql_cmd
+    final_cmd = []
+
+    cmd_init = "#{resource[:psql_path]} --tuples-only --quiet "
```

Comment:
`--no-psqlrc` is also always a good idea on scripts.  Otherwise the client's configuration can interfere.